Domains/Core/BisCore.ecschema.xml

```diff
@@ -566,14 +565,14 @@
 
     <ECEntityClass typeName="GeometryDetailRecord" modifier="Abstract" displayLabel="Geometry Detail Record" description="An abstract InformationRecordElement that holds geometry for use by the 'feature tree' stored in the `GeometricElement3d.GeometryOperations` property of its parent Element.">
         <BaseClass>InformationRecordElement</BaseClass>
-        <ECProperty propertyName="GeometryStream" typeName="binary" extendedTypeName="GeometryStream" displayLabel="Geometry Stream" description="Binary stream used to persist the geometry of this bis:Element.">
+        <ECProperty propertyName="GeometryMap" typeName="binary" extendedTypeName="GeometryMap" displayLabel="Geometry Map" description="Binary stream used to persist the geometries of this bis:Element.">

```

Comment:
Has Brien's code to work with GeometryDetailRecords also been updated to work with GeometryMap instead of GeometryStream?

Reply:
@bbastings Looks like there won't be any issues to update the code if/when we decide to merge that branch.

Reply:
Agree.